npm/angular/src/mount.ts

```diff
@@ -278,21 +278,22 @@ function setupComponent<T extends { ngOnChanges? (changes: SimpleChanges): void
  * @param {Type<T> | string} component Angular component being mounted or its template
  * @param {MountConfig<T>} config configuration used to configure the TestBed
  * @example
- * import { HelloWorldComponent } from 'hello-world/hello-world.component'
+ * import { mount } from '@cypress/angular'
+ * import { StepperComponent } from './stepper.component'
  * import { MyService } from 'services/my.service'
  * import { SharedModule } from 'shared/shared.module';
- * import { mount } from '@cypress/angular'
- * it('can mount', () => {
- *  mount(HelloWorldComponent, {
+ * it('mounts', () => {
+ *  mount(StepperComponent, {
```

Comment:
Let's properly indent these code blocks and add new lines between the imports. Also, we should drop the second argument for this example (keep it for the template example but it should just be `mount('<app-hello-world></app-hello-world>', { declarations: [HelloWorldComponent] })`. We don't need to show off the entire API int he example.